cohttp/src/cohttp.ml

```diff
@@ -2,7 +2,10 @@ module Accept = Accept
 module Auth = Auth
 module Body = Body
 module Conf = Conf
+
 module Connection = Connection
+[@@deprecated "Values of this type are useless and need to be used"]
```

Comment:
I don’t understand the deprecation comment here

Reply:
Whoops. It is fixed